cli/bin/env-setup

```diff
@@ -6,12 +6,8 @@ else
     BIN_DIR=$PWD/bin
 fi
 
+# real, absolute path to BIN_DIR
 FULL_BIN_PATH=$(python -c "import os; print(os.path.realpath('$BIN_DIR'))")
 
+# ensure BIN_DIR is prepended to PATH
 expr "$PATH" : "${FULL_BIN_PATH}.*" > /dev/null || export PATH=$FULL_BIN_PATH:$PATH
-export DCOS_CONFIG=~/.dcos/dcos.toml
-
-if [ ! -f "$DCOS_CONFIG" ]; then
```

Comment:
Once you've moved the creation block, please remove https://github.com/mesosphere/dcos-cli/blob/master/bin/install/install-dcos-cli.sh#L49-L50.